redash/query_runner/__init__.py

```diff
@@ -73,6 +73,23 @@ def annotate_query(cls):
     def configuration_schema(cls):
         return {}
 
+    def get_data_source_version(self):
+        if self.data_source_version_query is None:
+            raise NotImplementedError
+        data, error = self.run_query(self.data_source_version_query, None)
+        version = json.loads(data)['rows'][0]['version']
+        if(self.data_source_version_post_process == "split by space take second"):
+            version = version.split(" ")[1]
+        elif(self.data_source_version_post_process == "split by space take last"):
+            version = version.split(" ")[-1]
+        elif(self.data_source_version_post_process == "none"):
+            version = version
```

Comment:
This could be more Pythonic... no parens needed and add a space after the `if/elif`s.